qiskit/test/decorators.py

```diff
@@ -67,15 +68,62 @@ def slow_test(func):
 
     @functools.wraps(func)
     def _wrapper(*args, **kwargs):
-        skip_slow = not TEST_OPTIONS['run_slow']
-        if skip_slow:
+        if not TEST_OPTIONS['run_slow']:
             raise unittest.SkipTest('Skipping slow tests')
 
         return func(*args, **kwargs)
 
     return _wrapper
 
 
+def latex_test(func):
+    """Decorator that signals that the test requires LaTeX to run.
+
+    Args:
+        func (callable): test function to be decorated.
+
+    Returns:
+        callable: the decorated function.
+    """
+
+    @functools.wraps(func)
+    def _wrapper(*args, **kwargs):
+        if TEST_OPTIONS['force_visualization']:
```

Comment:
I'm not sure I understand why we need this argument. All it does is make a skip a failure, that doesn't seem to add much value. Especially when we change the test runner as suggested above stestr will treat a run with all skips as a failure (or any run without any successful tests).

This honestly is a use case where I wish we didn't force the use of stdlib unittest as the base class because every test library for python besides stdlib supports setting attributes on a test. With that we could be a lot more targeted in how we run tests (each test could say it's an extra viz test in an attribute and we only run those in the extra job). We could easily switch to using `testtools.TestCase` maintain the unittest compatibility (ie `python -m unittest discover` still works fine), which is important and something other popular test libs like (pytest and nose don't provide), but then it would enable setting attributes in tests and other advanced features like attachments. But we can discuss that separately since it's larger than this change